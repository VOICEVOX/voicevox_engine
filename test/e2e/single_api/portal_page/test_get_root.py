"""
/ API のテスト
"""

from fastapi.testclient import TestClient
from syrupy.assertion import SnapshotAssertion


def test_get_root_200(client: TestClient, snapshot: SnapshotAssertion) -> None:
    response = client.get("/")
    assert response.status_code == 200
    assert snapshot == response.content.decode("utf-8")
