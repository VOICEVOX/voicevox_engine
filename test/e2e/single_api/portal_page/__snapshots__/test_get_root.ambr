# serializer version: 1
# name: test_get_root_200
  '''
  
          <html>
              <head>
                  <title>VOICEVOX ENGINE OSS</title>
              </head>
              <body>
                  <h1>VOICEVOX ENGINE OSS</h1>
                  VOICEVOX ENGINE OSS へようこそ！
                  <ul>
                      <li><a href='/setting'>設定</a></li>
                      <li><a href='/docs'>API ドキュメント</a></li>
          </ul></body></html>
          
  '''
# ---
